redcap_bridge/cli.py

```diff
@@ -28,18 +31,28 @@ def main(command_line=None):
                           help="Format to store the data (json/csv)")
     download.add_argument("-c", "--compressed", action='store_true',
                           help="Compress the output file (use labels and merge checkbox columns)")
+    download.add_argument("--server", type=str, nargs=1, metavar='server',
+                          help="Server name")
```

Comment:
based on this definition the user does not have any clue about accepted values here. Is it possible to define this to accept only two distinct strings already using argparse? Otherwise this definitely needs to go into the `help` string.

Reply:
Also the default value would be good to document here.

Reply:
I think that was called `choices`...